tests/core/test_linalg.py

```diff
@@ -215,7 +228,8 @@ def test_rr(app_inst: ArrayApplication):
     app_inst = conftest.get_app("ray")
     # test_inv_assumptions(app_inst)
     # test_inv(app_inst)
-    test_qr(app_inst)
+    test_inv_uppertri(app_inst)
+    # test_qr(app_inst)
     # test_svd(app_inst)
```

Comment:
I was wondering why these other tests are commented out. @elibol

Reply:
I believe it is commented out for testing locally. `pytest` should run all the methods with `test_` in the build checks.